apps/tx-builder/src/pages/ReviewAndConfirm.tsx

```diff
@@ -0,0 +1,121 @@
+import { Button, FixedIcon, Title } from '@gnosis.pm/safe-react-components';
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+
+import DeleteBatchModal from '../components/modals/DeleteBatchModal';
+import TransactionsBatchList from '../components/TransactionsBatchList';
+import useModal from '../hooks/useModal/useModal';
+import { HOME_PATH } from '../routes/routes';
+import { ProposedTransaction } from '../typings/models';
+import { useEffect } from 'react';
+
+type ReviewAndConfirmProps = {
+  transactions: ProposedTransaction[];
+  handleSubmitTransactions: () => void;
+  handleRemoveTransaction: (index: number) => void;
+  handleRemoveAllTransactions: () => void;
+  handleReorderTransactions: (sourceIndex: number, destinationIndex: number) => void;
+};
+
+const ReviewAndConfirm = ({
```

Comment:
This is the new Review and Confirm Screen component:
![Captura de pantalla 2022-02-24 a las 11 55 51](https://user-images.githubusercontent.com/26763673/155511321-f37cc5a4-2574-4e8b-a893-a1fec57fbb36.png)